src/tests/ftest/osa/offline_drain.yaml

```diff
@@ -54,8 +54,8 @@ pool:
 container:
     type: POSIX
     control_method: daos
-    oclass: RP_2G1
-    properties: cksum:crc64,cksum_size:16384,srv_cksum:on
+    oclass: RP_3G6
```

Comment:
Why specifically RP_3G6? Why not RP_3GX?

Reply:
Just using similar object class as offline reintegration test?

Reply:
And why does that one also use RP_3G6? I'm just trying to understand for my sake :)

Reply:
This was updated by Fan Yong when we hit the redundancy factor problem under the offline_reintegration test case. I believe this is one of the default replicated objects defined in src/include/daos_obj_class.h. 
3GX--> normally the object is replicated to every target. Initially, we had issues with 2GX, 3GX object class with jump placement algorithm not handing certain condition, Probably, it is fixed now. Sometimes, the developers recommend using numbers like 2G8,3G6, etc instead of 2GX, 3GX.